### ISSUE TITLE
Read remote server

### ISSUE THREAD

**Issue by maitrepylos** [lino-framework/eidreader#6]
Hello,
I test your EID script and thank you for this library.
It works very well locally.
How can I install this library on a remote server and read the ID card on my machine? I didn't understand if it was possible ?
Thank you very much.

**Comment by lsaffre** [comment]
Thanks for reporting your issue. This library runs only on the client. You must yourself add functionality to your web application so that your web page (a) invokes eidreader on the client using an a href tag with a custom URL protocol. And then your server must accept the POST requests issued by eidreader and process that data. See https://eidreader.lino-framework.org/usage.html#the-web-application

**Comment by TitiBordon** [comment]
@lsaffre 
Hello. So, if I understand you, we can't install this on the server? 
We must install on all clients (we have over 2000 pcs that can using the app).
My app is on a linux server (which contains the apps for our company).
But we lanch these via a firefox browser in windows10 (client machine).

I've set up drag&drop of the photo using the eid viewer and that's work very well but my chief doesn't want use the drag&drop, he wants it to be done automatically by clicking on a link or a button.

So it's not possible unless you install it on every PC?

**Comment by lsaffre** [comment]
Yes, you must install something on the client machine. I don't know any method around this. It's not my decision. If a browser would send data from a smart card reader to an arbitrary web server without installing some plugin or extension, you would consider this a security hole. 
But if you have over 2000 clients, don't you use some configuration management system for deploying upgrades and new software on these pcs? Can't you do the installation centrally once for all clients? 
https://eidreader.lino-framework.org/install.html
